Installer/MainInstaller.sh

```diff
@@ -1,6 +1,6 @@
 DIR=`pwd`
 
-function finish {
+finish() {
```

Comment:
MainInstaller.sh wasn't running on my system. Said it couldn't find the function, changing this fixed it for me. Not sure why it wasn't working, since I had ran the script with no issues before